primitives/io/src/lib.rs

```diff
@@ -1134,6 +1135,45 @@ pub trait Sandbox {
 	}
 }
 
+/// Runtime spawn extension.
+pub trait RuntimeSpawn : Send {
```

Comment:
Not sure what you mean. It is used for an extension below.

Reply:
Just pointing out that we got a single implementation behind a dyn pointer in the RuntimeSpawnExt, so I was thinking it could be remove (except if their could be other future implementation).

Reply:
Removed and replaced with what?

Reply:
`Box<dyn RuntimeSpawn>` by `RuntimeInstanceSpawn` directly.
But I just realized that this will mess up the crate hierarchy.

Reply:
Yeah, exact type can't be known in `sp-io`